wordpress-mobile/WordPress-iOS
# Moved AccountSettingsRemote + tests + basic models to WPKit framework

Fixes #7304 — This PR migrates: 

* The `AccountSettingsRemote ` remote service class 
* Basic (non-managed) models related to 👆 
* Related unit tests + mock data

to the WordPressKit framework. Also, ref #7303 & #7301

**To test:**

1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run all unit tests and make sure they are green
5. Run the app and log in with a .com account. Load and update a few account settings (e.g. display name) and ensure data is fetched and saved properly.

Needs review: @astralbodies
